idea_maluuba_formatter.xml

```diff
@@ -1,9 +1,9 @@
 <code_scheme name="MaluubaStyle">
   <option name="OTHER_INDENT_OPTIONS">
     <value>
-      <option name="INDENT_SIZE" value="4" />
-      <option name="CONTINUATION_INDENT_SIZE" value="8" />
-      <option name="TAB_SIZE" value="4" />
+      <option name="INDENT_SIZE" value="2" />
```

Comment:
The indention scheme is 4, 4, 8 in the code style page, when loading this file, and is 8, 8, 16 when loading the previous version of the file - it seems 2, 2, 4 is the proper xml setting to get 4, 4, 8 in the IDE.